daprdocs/content/en/developing-applications/building-blocks/pubsub/howto-route-messages.md

```diff
@@ -143,6 +143,34 @@ app.listen(port, () => console.log(`consumer app listening on port ${port}!`))
 ```
```

Comment:
on line 137 (for some reason I can't add an in-line comment there) - should this remain as `'/dsstatus'` or be changed to `'/withdraw'`?